Mathlib/MeasureTheory/Integral/IntervalIntegral.lean

```diff
@@ -971,6 +971,18 @@ theorem integral_Iic_sub_Iic (ha : IntegrableOn f (Iic a) μ) (hb : IntegrableOn
   exacts [measurableSet_Ioc, ha, hb.mono_set fun _ => And.right]
 #align interval_integral.integral_Iic_sub_Iic intervalIntegral.integral_Iic_sub_Iic
 
+theorem integral_Iic_add_Ioi (h_left : IntegrableOn f (Iic b) μ)
+    (h_right : IntegrableOn f (Ioi b) μ) :
```

Comment:
Which is usually preferred between (h_left : IntegrableOn f (Iio b) μ) (h_right : IntegrableOn f (Ici b) μ) and Integrable f μ?

Reply:
I don't know, but I suspect it might be slightly easier to restrict integrability to the two subset then show that integrability on the enitre domain follows from the integrability on the two subsets. 
I basically did what is done in `integral_Iic_sub_Ioi` a few lines above.